include/pipettes/firmware/interfaces_g4.hpp

```diff
@@ -0,0 +1,126 @@
+#pragma once
+
+#include "motor-control/core/motor_messages.hpp"
+#include "motor-control/core/stepper_motor/motor_interrupt_handler.hpp"
+#include "motor-control/firmware/stepper_motor/motor_hardware.hpp"
+#include "pipettes/core/gear_motor_tasks.hpp"
+#include "pipettes/core/interfaces.hpp"
+#include "pipettes/core/linear_motor_tasks.hpp"
+#include "pipettes/core/motor_configurations.hpp"
+#include "pipettes/core/pipette_type.h"
+#include "pipettes/firmware/pipette_motor_hardware.hpp"
+
+#pragma GCC diagnostic push
+// NOLINTNEXTLINE(clang-diagnostic-unknown-warning-option)
+#pragma GCC diagnostic ignored "-Wvolatile"
+#include "motor_timer_hardware.h"
+#pragma GCC diagnostic pop
+
+/*
+ * (CM, 11/3/2022)
+ * This file exists in addition to interfaces.hpp to accommodate the l5 firmware
+ * after the changes in the enable-pipette-encoder-output branch. interfaces.hpp
+ * can be deleted with the l5 firmware.
```

Comment:
Can you add in this comment that the `interfaces_g4.hpp` should be renamed `interfaces.hpp` when this happens?